### PR TITLE
Strict mode for ponte

### PR DESCRIPTION
**Pull request by psorowka**
This Pull Requests introduces strict mode for ponte and fixes three assignment/declaration issues (one of them critical).

Limited Travis CI and recommended version to 5.5 as this is the last version mosca runs on reasonably and coap fails on 5.7 currently.

### CODE REVIEW

**Comment by psorowka** [comment]
errr. email address had changed. correct this in my eclipse account, should I reopen that?

**Comment by mcollina** [comment]
ip validation still fails :( https://dev.eclipse.org/eclipse-webhook/services/status_details.php?id=56d2d8a2b70df

**Comment by psorowka** [comment]
oops, i obviously didnt complete the address update... should work now.

@mcollina do you know how to trigger the ip check again?

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ language: node_js
 node_js:
   - 0.12
   - 4
-  - 5
+  - 5.5
```
Comment (mcollina) [inline .travis.yml]: let's keep 5.5 and 5.6 here. 5.7.1 with the udp fix will come out this week.

**Comment by mcollina** [comment]
Do a commit or force-push.

File: .travis.yml (hunk 1 above)
Comment (psorowka) [inline .travis.yml]: I removed 5.6 because of https://github.com/mcollina/mosca/issues/412

Is this no issue anymore?

Comment (mcollina) [inline .travis.yml]: that is solved, it was a `bl` problem.

**Comment by psorowka** [comment]
sorry i have to work on the ip validation.. complicated stuff :)

**Comment by psorowka** [comment]
closing this in favor of #22 i didnt manage to clean up the signoff mess here...

**Pull request closed without merging**